doc/protocol/README.md

```diff
@@ -0,0 +1,18 @@
+# dGrants Protocol
+
+The dGrants protocol specification
+
+## Directory Structure
+
+.
+├── protocol.md                   # dGrants protocol definition
```

Comment:
I don't see a `protocol.md` folder

Reply:
I hadn't added the doc yet but went ahead and added the start of protocol.md.